content/practices/event-storming-process-modelling/index.md

```diff
@@ -0,0 +1,151 @@
+   To save time you can identify only the grammar that is relevant to the discussion, you do not need to find all grammar for all events.
+
+1. Review Questions and Pain Points: As with he formal method try to answer as many questions as possible and rationalize pain points and opportunities
+
+#### Success/Expected Outcomes
+   Generally you are looking for the same types of outcome as the formal method, however the sessions tend to be shorter and more focused on the flow of the process, and so only Events and Integrations are required
+
+#### Facilitator Notes & Tips
+
+Events: Finding suitable events is important to getting a good architecture. 
+ - Identify the important state change and write the event to make the subject of the event, the object that changed state. i.e. the "Customer created an order", in this sentence the state of the Order is more important so we re-write it as "Order was created". Another might be "Customer Changed Address" In this instance the change of state to the customer's address is important, so the event could be written as "Customer's Address Changed". 
+ - Try to avoid "Passive-Aggressive Commands" as described in Martin Fowler's blog [What do you mean by “Event-Driven”?](https://martinfowler.com/articles/201701-event-driven.html). These are often found when the object raising state changes is a listener to that event, in order to complete a process. One way to handle this could be to squash all policies, commands and events into a new policy, command, written to describe the whole process squashed. Identify which events are listened to and what negative events might be raised as a result of any failures from the squashed commands, again only if they are listened to.
+
+Policies: Writing Policies can be important to ensure you have the correct flow and capturing the correct business requirement
+ - Policies should be written as a business decision that guides an actor or system to the correct action. The actor may be responsible for implementing the policy but they may not be responsible for defining the policy. If good Roles are identified in the `EventStorming` stage then you should be able to identify roles that influence that decision.
+
+Thin Slices: Thin Slices can be heavily influenced by the goals of your project, but a good start might be to pick out a `golden path` and reduce it to the least set of events required to achieve the final outcome.
+
+## Real World Examples
+![This is the placeholder for sample 1](/images/practices/event-storming-process-modelling/SampleEventStormingAndModellingBoard.JPG)
+![This is the placeholder for sample 2](/images/practices/event-storming-process-modelling/SampleProcessModellingBoard.JPG)
+![This is the placeholder for sample 3](/images/practices/event-storming-process-modelling/SampleSoftwareModellingBoard.JPG)
+![This is the placeholder for sample 3](/images/practices/event-storming-process-modelling/SampleProcessModellingAndBorisBoard.JPG)
+## Recommended Reading
```

Comment:
I would highly recommend to add a link to the official DDD community on GitHub.

https://github.com/ddd-crew/ddd-starter-modelling-process